doc/source/whatsnew/v0.16.2.txt

```diff
@@ -25,6 +25,8 @@ New features
 Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
+- Removed duplicate scroll bars in DataFrame HTML representation for IPython 3.0.0 or later. (:issue:`10231`)
```

Comment:
This needs some addtl explanation w.r.t. ``max_rows``. IOW, I find that using ``pd.set_setoption('max_rows,None)`` is now a nice way to use the notebook (for a terminal I have this set to a smaller value, however). maybe mention of same in ``options.rst``